blocks/marketo-form/marketo-form.js

```diff
@@ -0,0 +1,21 @@
+function addScript(src) {
+  const script = document.createElement('script');
+  script.src = src;
+  document.head.appendChild(script);
+}
+// creation of block for marketo form
+export default function decorate(block) {
+  addScript('//899-BTB-436.mktoweb.com/js/forms2/js/forms2.min.js');
+  [...block.children].forEach((row) => {
+    const section = row.querySelector('p');
+    const divId = section.innerHTML;
+    const idstringValue = divId.split('_')[1];
+    const cForm = document.createElement('form');
```

Comment:
add class="marketoform" to form in js. while write scss use selector "form.marketoform" to make it more specific.

Remove importants from scss.